docs/openapi-docs/src/main/scala/sttp/tapir/docs/openapi/Extensions.scala

```diff
@@ -0,0 +1,12 @@
+package sttp.tapir.docs.openapi
+
+import sttp.tapir.Extension
+import sttp.tapir.apispec.ExtensionValue
+import sttp.tapir.internal.IterableToListMap
+
+import scala.collection.immutable.ListMap
+
+object Extensions {
```

Comment:
I suppose this can be `private[openapi]` - it's used only internally, right?

Reply:
It's used internally, yes. I think it is possible that someone who will modify OpenAPI object manually may need it, but maybe it's not the reason to keep it public, not sure. Changed to private for now.